.github/workflows/comprehensive.yml

```diff
@@ -0,0 +1,442 @@
+        echo ------------ lsb_release - a
+        lsb_release -a || true
+        echo ------------ hostnamectl
+        hostnamectl || true
+        echo ------------ /etc/os-release
+        cat /etc/os-release || true
+        echo ------------ /proc/version
+        cat /proc/version || true
+        echo ------------ lscpu
+        lscpu || true
+
+    - name: UPDATE - system deps and install libc6, net-tools, btrfs-progs, zip, unzip, bash, procps
+      run: |
+        sudo apt-get update
+        sudo apt-get install libc6 net-tools btrfs-progs zip unzip
+        sudo apt-get install bash
+        sudo apt-get install procps
+
+    - name: INIT - install docker with compose
+      run: |
+        echo " --------------------------- remove old packages ------------------------------------------------------------------------------------------------------ "
+        sudo apt-get remove docker* containerd* || true
+        echo " --------------------------- install containerd.io ---------------------------------------------------------------------------------------------------- "
+        sudo apt-get install containerd.io
+        echo " --------------------------- install docker.io -------------------------------------------------------------------------------------------------------- "
+        echo " --------------------------- install docker-compose --------------------------------------------------------------------------------------------------- "
+        sudo apt-get install docker-compose
+
+    - name: INIT - reconfigure docker and restart its daemon
+      run: |
+        sudo systemctl unmask docker
+        sudo groupadd docker || true
+        sudo gpasswd -a username docker || true
+        sudo service docker restart || true
+
+    - name: INIT - uninstall old Node JS if any
+      run: |
+        sudo npm cache clean -f || true &> /dev/null
+        sudo apt-get remove nodejs npm node -y || true &> /dev/null
+        sudo apt-get purge nodejs -y || true &> /dev/null
+        sudo apt-get autoremove -y || true &> /dev/null
+        sudo rm -rf /usr/bin/node || true &> /dev/null
+        sudo rm -rf /usr/include/node || true &> /dev/null
+        sudo rm -rf /usr/lib/node_modules || true &> /dev/null
+        sudo rm -rf /usr/local/bin/npm  || true &> /dev/null
+        sudo rm -rf /usr/local/share/man/man1/node* || true &> /dev/null
+        sudo rm -rf /usr/local/lib/dtrace/node.d || true &> /dev/null
+        sudo rm -rf ~/.npm || true &> /dev/null
+        sudo rm -rf ~/.node-gyp || true &> /dev/null
+        sudo rm -rf /opt/local/bin/node || true &> /dev/null
+        sudo rm -rf /opt/local/include/node || true &> /dev/null
+        sudo rm -rf /opt/local/lib/node_modules || true &> /dev/null
+        sudo rm -rf /usr/local/lib/node* || true &> /dev/null
+        sudo rm -rf /usr/local/include/node* || true &> /dev/null
+        sudo rm -rf /usr/local/bin/node* || true &> /dev/null
+
+    - name: INIT - install Node JS
```

Comment:
There is a github action for doing this:
`actions/setup-node`

Reply:
If you cannot install node then you should not use it at all. Github can go to hell in this case.